src/main/scala/org/zalando/kanadi/api/Subscriptions.scala

```diff
@@ -1276,7 +1276,9 @@ case class Subscriptions(baseUri: URI, oAuth2TokenProvider: Option[OAuth2TokenPr
               streamConfig,
               modifySourceFunction
             ))
-        }
+        } else
+          logger.info(
+            s"Nakadi stream cancelled by the client application, not reconnecting Old StreamId: ${connectionClosedData.oldStreamId}, SubscriptionId: ${subscriptionId.id.toString}")
```

Comment:
toString is redundant

Reply:
If the type is not `String` I prefer to explicitly call `.toString` so I am more aware of the types (in this case subscription id is a UUID)